python/llm/example/transformers/transformers_int4/chatglm/README.md

```diff
@@ -12,27 +12,17 @@ To run these examples with BigDL-LLM, we have some recommended requirements for
 ## Example: Predict Tokens using `generate()` API
 In the example [generate.py](./generate.py), we show a basic use case for a ChatGLM model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations.
 ### 1. Install
-We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](../README.md#conda-installation).
 
 After installing conda, create a Python environment for BigDL-LLM:
 ```bash
 conda create -n llm python=3.9 # recommend to use Python 3.9
 conda activate llm
 
-pip install bigdl-llm[all] # install bigdl-llm with 'all' option
+pip install --pre --upgrade bigdl-llm[all] # install bigdl-llm with 'all' option
 ```
 
 ### 2. Run
-```
```

Comment:
We should add a short sentence of description here: e.g.

After environment setup, you could run the example [generate.py](./generate.py).

Something like this, but you could optimize the sentence :)